resources/views/components/head-analytics.blade.php

```diff
@@ -35,6 +35,8 @@
 
     $props = [
         'isAuthenticated' => auth()->check(),
+        'scheme' => request()->cookie('scheme') ?: 'dark',
+        'theme' => request()->cookie('theme') ?: 'default',
```

Comment:
I don't think it hurts anything, but these values are still sent for the /manage/ pages, which have their own color theme settings.

Reply:
Doesn't hurt anything, but a good callout nonetheless & I appreciate it.

Since I'm interested in the cookie value itself, I don't believe this will lead to any data poisoning.